notebooks/hello-npu/hello-npu.ipynb

```diff
@@ -0,0 +1,1253 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    %pip install -q "openvino-dev&gt;=2024.1.0"
</pre><p>list of dependencies looks strange... in code below, torch and torchvision used for mode conversion, it does not looks like tensorflow is required to run this notebook.</p><p>Also we avoid to use openvino-dev as it contains deprecated tools (model_optimizer), but code bellow used benchmark app and convert_model that are part of openvino package.</p><p> </p><p>Also openvino 2024.1 is not released, but I believe, it is possible to launch this notebook using openvino-nightly, right?</p><p>Please use openvino-nightly for unblocking notebook merge before release package will be published </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1898/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>